src/components/BodyWithResults/BodyWithResults.tsx

```diff
@@ -20,15 +21,28 @@ export default React.memo((props: Props) => {
         />
     ));
 
+    const getResultList = () => {
```

Comment:
Rendering entities names should start from 'render', like 'renderResultList'. There is also no sense in making function here. JSX syntax is just synthetic sugar using `<Component>` as `Component()` under the hood. So you can just take assign your JSX to the var and output it later like `{resultsList}`.

Reply:
renamed to renderResultList

Reply:
Still using function call instead of returning render result.